mod/wikirate_csv_import/lib/csv_row/structure/answer_csv.rb

```diff
@@ -66,7 +66,7 @@ def check_existence_and_type name, type_id, type_name=nil
         end
       end
 
-      def resolve_source_duplicates existing_source_card
+      def resolve_source_duplication existing_source_card
```

Comment:
please confirm that this change is correct.  `resolve_source_duplicates` is never called. Looked to me like this was intended to be an override.

Reply:
to me too